packages/react-server/core/FramebackController.js

```diff
@@ -228,40 +229,19 @@ class FramebackController extends EventEmitter {
 
 	createFrame(url){
 
-		this.frame = document.createElement("iframe");
+		const frame = this.frame = document.createElement("iframe");
```

Comment:
Yep, it's doing the same stuff in a different way.  Each browser has a different set of implementation quirks.  Last October one such FF quirk caused the introduction of the awkward unload/load dance (see `RED-70527` for details).  That doesn't _seem_ to be necessary anymore.  Or else I've settled on a _different_ simple implementation than we were on previously that happens to also work in FF.

Or else I haven't tickled the repro case for the FF quirk correctly and this is going to bounce right back to me. :p